lib/Echidna/Campaign.hs

```diff
@@ -45,13 +45,16 @@ import Echidna.Exec
 import Echidna.Solidity
 import Echidna.Test
 import Echidna.Transaction
+import Echidna.Mutator
 
 instance MonadThrow m => MonadThrow (RandT g m) where
   throwM = lift . throwM
 instance MonadCatch m => MonadCatch (RandT g m) where
   catch = liftCatch catch
 
-type MutationConsts = (Integer, Integer, Integer)
+type MutationConsts = (Integer, Integer, Integer, Integer, Integer)
+defaultMutationConsts :: MutationConsts 
+defaultMutationConsts = (10,1,1,1,1)
```

Comment:
Is it possible to name somehow those numbers? I see how they are used in `seqMutators` but it feels a bit arbitrary

Reply:
This distribution is a little arbitrary in `seqMutators`. My idea is to experiments with these as a vector/list and leave for a future PR naming the parameters that are actually useful and setting a reasonable default.